designs/gpu-selection-and-bootstrap.md

```diff
@@ -108,10 +109,10 @@ Here are some relevant labels:
 
 | Label                                 | Description                               |
 |---------------------------------------|-------------------------------------------|
-| `karpenter.k8s.azure/sku-family`       | Family of the SKU (e.g., standardAv2Family)|
-| `karpenter.k8s.azure/sku-subfamily`    | Subfamily of the SKU                      |
-| `karpenter.k8s.azure/sku-cpu`          | Number of virtual CPUs                    |
-| `karpenter.k8s.azure/sku-accelerator`  | Type of accelerator (e.g., Nvidia)        |
+| `karpenter.azure.com/sku-family`       | Family of the SKU (e.g., standardAv2Family)|
+| `karpenter.azure.com/sku-subfamily`    | Subfamily of the SKU                      |
```

Comment:
We don't support subfamily

Reply:
removed, although I think we should have a conversation of supporting it for gpu